Formula/mosdepth.rb

```diff
@@ -20,12 +19,6 @@ class Mosdepth < Formula
 
   def install
     bin.install "mosdepth"
-    unless OS.mac?
-      system "patchelf",
-        "--set-interpreter", HOMEBREW_PREFIX/"lib/ld.so",
-        "--set-rpath", HOMEBREW_PREFIX/"lib",
-        bin/"mosdepth"
-    end
```

Comment:
Why did you remove this chunk?

Reply:
It caused a build error saying it was a static binary, see https://github.com/brewsci/homebrew-bio/runs/605628184

Reply:
Got it. Thanks!